announcements/2020-Week-46/announcement.md

```diff
@@ -0,0 +1,13 @@
+# :zap: This Week in Provider-Independent Announcements (November 13, 2020):zap:
+
+## Giant Swarm Catalog (Managed Apps)
+
+1. [Cert Manager v2.3.2](https://github.com/giantswarm/cert-manager-app/blob/master/CHANGELOG.md#232---2020-11-09) upgrades to upstream v1.0.4 and adds small improvements.
```

Comment:
@glitchcrab Are the other changes noteworthy?

Reply:
It does add the values schema so it may well be worth mentioning that.

Reply:
lol i made that change of course i didn't think it's important hahaha